src/video/wayland/SDL_waylandvulkan.c

```diff
@@ -41,6 +41,13 @@
 #define DEFAULT_VULKAN "libvulkan.so.1"
 #endif
 
+SDL_ELF_NOTE_DLOPEN(elfnote_wayland,
```

Comment:
Not `elfnote_wayland_vulkan`?

Reply:
That's more correct! Thanks!
```suggestion
SDL_ELF_NOTE_DLOPEN(elfnote_wayland_vulkan,
```